example_cp.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Example of CellProfiler project for idrstream
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+import shutil
+
+from cellpose import core
+
+from idrstream.idr_stream import IdrStream
+
+
+# ## Initialize idrstream
+
+# In[2]:
+
+
+stream_type = "CP"
+idr_id = "idr0013"
+tmp_dir = pathlib.Path("tmp/")
+final_data_dir = pathlib.Path("mitocheck_control_features/")
+try:
+    shutil.rmtree(tmp_dir)
+    # uncomment the line below if you would like to remove the final data directory (e.g. all .csv.gz files)
+    # shutil.rmtree(final_data_dir)
+    pass
+except:
+    print("No files to remove")
+
+stream = IdrStream(stream_type,idr_id, tmp_dir, final_data_dir, log='idr_stream.log')
+
+
+# ## Initialize CellProfiler metadata compiler
+
+# In[3]:
+
+
+data_to_process_tsv = pathlib.Path("example_files/data_to_process.tsv")
```

Comment:
Later in this file `data_to_process` gets loaded in but then immediately modified with `data_to_process[data_to_process["Well"] != "A1"]`. Should this also be done before `stream.init_cp_metadata(data_to_process_tsv)`?

Reply:
@roshankern 

By what I understand, `data_to_process.tsv` gets loaded in as to be a variable in run_stream so that the correct images from the movies are downloaded. 

Since I only need the file for CellProfiler to connect the images to the correct metadata, then it doesn't matter if the file as the A1 well because the images will never be downloaded. 

As well, even if I did that before converting the .tsv file to .csv, that code will not change and remove A1 from the .tsv file in the `/example_files` folder, it only changes it in the dataframe in the notebook. So the .csv file created would not have this change anyway.